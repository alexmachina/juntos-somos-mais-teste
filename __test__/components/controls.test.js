import React from "react";
import Controls from "../../pages/customers/list/components/controls";
import renderer from "react-test-renderer";

describe("Components", () => {
  describe("Controls", () => {
    it("Renders correctly", () => {
      const tree = renderer.create(<Controls />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
