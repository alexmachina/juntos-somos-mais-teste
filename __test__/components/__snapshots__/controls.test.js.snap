// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Controls Renders correctly 1`] = `
<div
  className="MuiGrid-root makeStyles-container-1 MuiGrid-container MuiGrid-justify-xs-center"
>
  <div
    className="MuiGrid-root makeStyles-stepperContainer-2 MuiGrid-item MuiGrid-grid-xs-12"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation0 MuiMobileStepper-root MuiMobileStepper-positionStatic"
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M15.41 16.59L10.83 12l4.58-4.59L14 6l-6 6 6 6 1.41-1.41z"
            />
          </svg>
          Back
        </span>
      </button>
      <div
        className="MuiMobileStepper-dots"
      >
        <div
          className="MuiMobileStepper-dot MuiMobileStepper-dotActive"
        />
      </div>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Next
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </span>
      </button>
    </div>
  </div>
</div>
`;
